### DISCUSSION TITLE
Import from Gallery 3?

### DISCUSSION THREAD

**Discussion by YouveGotMeowxy** [photoprism/photoprism#1240] [Q&A] [answered]
I'm looking for a modern replacement for [Gallery 3](http://galleryproject.org/), but have an  extensive library with everything meticulously tagged, and comments on images, etc., and was hoping to find an app that can directly import everything from G3 so that it's basically a "drop in replacement" is it possible with Photoprism?

**Comment by graciousgrey** [comment, marked as answer]
@YouveGotMeowxy depends on how Gallery 3 saves all the metadata like tags or comments. PhotoPrism reads many metadata tags. You find an overview here: https://docs.photoprism.org/user-guide/advanced/metadata/
